run/test/specs/input_validations/onboarding_no_name.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { androidIt, iosIt } from '../../../types/sessionIt';
+import * as onboarding from '../locators/onboarding';
+import { openAppOnPlatformSingleDevice, SupportedPlatformsType } from '../utils/open_app';
+import { localize } from '../../../localizer/i18n/localizedString';
+
+iosIt('Onboarding no name', onboardingNoName);
+androidIt('Onboarding no name', onboardingNoName);
+
+// checking that we're not trying to test with a non-empty name
+const emptyName = '';
+if (emptyName.length > 0) {
```

Comment:
Again no need to test our own test inputs

Reply:
removed

Reply:
Happy for you to add this back in, just move it into the test